include/boost/gil/extension/dynamic_image/any_image_view.hpp

```diff
@@ -135,15 +136,18 @@ struct dynamic_x_step_type<any_image_view<Views>>
     using dynamic_step_view = typename gil::dynamic_x_step_type<T>::type;
 
 public:
-    using type = any_image_view<mp11::mp_transform<dynamic_step_view, Views>>;
+    using type = mp11::mp_rename<
+        mp11::mp_transform<dynamic_step_view, typename any_image_view<Views...>::mp_views_t>,
```

Comment:
This is unnecessary. mp_transform will work on any_image_view<Views...> directly. No need to go to mp_list and then back.

Reply:
Thanks! Much cleaner.